composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "gitonomy/gitlib",
+    "name": "iwazaru/gitlib",
```

Comment:
:-1: This is also not actually needed if you want to install your fork anyway. In fact, publishing forks on packagist is a bad practice.